src/main/java/Wolox/training/models/User.java

```diff
@@ -0,0 +1,118 @@
+package Wolox.training.models;
+
+import Wolox.training.commons.Constants;
+import Wolox.training.exceptions.BookAlreadyOwnedException;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+import org.springframework.http.HttpStatus;
+
+@Entity
+@Table(name = "users")
+public class User {
+
+    public User() {
+    }
+
+    /**
+     * Primary key of User entity
+     */
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE)
+    private Long id;
+    @NotNull
+    @Column(nullable = false)
+    private String userName;
+    @NotNull
+    @Column(nullable = false)
+    private String name;
+    @NotNull
+    @Column(nullable = false)
+    private LocalDate birthdate;
+    @NotNull
+    @Column(nullable = false)
+    @ManyToMany(cascade = CascadeType.ALL)
```

Comment:
use MERGE and/or REFRESH, you can read more in this link https://www.baeldung.com/jpa-cascade-types